assets/js/src/newsletters/types.jsx

```diff
@@ -39,6 +39,13 @@ class NewsletterTypes extends React.Component {
             role="button"
             tabIndex={0}
             disabled={!window.mailpoet_premium_active}
+            onKeyDown={(event) => {
```

Comment:
I am just curious if this will trigger `this.onClick` even if the button is `disabled`.

Reply:
I cannot make it trigger for disabled buttons. Because I didn't manage to set the focus for disabled buttons.